dali/test/python/test_operator_uniform.py

```diff
@@ -27,14 +27,13 @@ def test_uniform_default():
         pipe.set_outputs(dali.fn.uniform(shape=[1e6]))
     pipe.build()
     oo = pipe.run()
-    possibly_uniform_distribution = oo[0].as_array()[0]
+    data = oo[0].as_array()[0]
     # normalize to 0-1 range
-    possibly_uniform_distribution_kstest = (possibly_uniform_distribution + 1) / 2
-    _, pv = st.kstest(rvs=possibly_uniform_distribution_kstest, cdf='uniform')
-    assert pv > 0.05, "`possibly_uniform_distribution` is not a uniform distribution"
-    assert (-1 <= possibly_uniform_distribution).all() and \
-            (possibly_uniform_distribution < 1).all(), \
-                "Value returned from the op is outside of requested range"
+    data_kstest = (data + 1) / 2
+    _, pv = st.kstest(rvs=data_kstest, cdf='uniform')
+    assert pv > 0.03, "`data` is not a uniform distribution"
+    assert (-1 <= data).all() and (data < 1).all(), \
+        "Value returned from the op is outside of requested range"
 
 def test_uniform_continuous():
```

Comment:
Maybe something like this would be better:
```
def test_uniform_continuous():
    results = np.array([])
    for _ in range(iters):
        pipe = dali.pipeline.Pipeline(1, 1, 0)
        lo = -100
        hi = 100
        test_range = (hi - lo) * np.random.random_sample(2) + lo  # 2 elems from [-100, 100] range
        test_range.sort()
        test_range = test_range.astype('float32')
        with pipe:
            pipe.set_outputs(dali.fn.uniform(range=test_range.tolist(), shape=[100]))
        pipe.build()
        oo = pipe.run()
        possibly_uniform_distribution = oo[0].as_array()[0]
        # normalize to 0-1 range
        possibly_uniform_distribution_kstest = (possibly_uniform_distribution - test_range[0]) / (test_range[1] - test_range[0])
        _, pv = st.kstest(rvs=possibly_uniform_distribution_kstest, cdf='uniform')
        assert (test_range[0] <= possibly_uniform_distribution).all() and \
                (possibly_uniform_distribution < test_range[1]).all(), \
                    "Value returned from the op is outside of requested range"
        results = np.append(results, pv)
    assert (results > 0.05).any(), "`possibly_uniform_distribution` is not a uniform distribution"
    assert (results > 0.03).all(), "`possibly_uniform_distribution` is not a uniform distribution"
```

Reply:
I changed to take the average of 10 iterations. Could you please check?

Reply:
I'm not sure if we need 1e6 sample now. Maybe 100 would do as well?